src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createcase/CreateCaseMidEventHandlerTest.java

```diff
@@ -64,6 +66,30 @@ void canHandleTest(EventType eventType) {
         assertTrue(midEventHandler.canHandle(MID_EVENT, callback));
     }
 
+    @ParameterizedTest
+    @EnumSource(value = EventType.class, names = {
+        "VALID_APPEAL_CREATED",
+        "NON_COMPLIANT",
+        "INCOMPLETE_APPLICATION_RECEIVED",
+        "CASE_UPDATED"
+    })
+    void canHandleTest_caseCreateBenefitSelectionFromList(EventType eventType) {
+        DynamicList expectedList = new DynamicList(
+            new DynamicListItem(IBCA_BENEFIT_CODE, INFECTED_BLOOD_COMPENSATION), new ArrayList<>());
```

Comment:
does this reflect the benefit list as it is displayed on the manual case create page? Check getBenefitDescriptions in SSCS Util to see how that list is generated